Assets/Scripts/Game/UserInterfaceWindows/DaggerfallExteriorAutomapWindow.cs

```diff
@@ -453,6 +457,19 @@ protected override void Setup()
             isSetup = true;
         }
 
+        protected virtual void SetupCaptionColors(Texture2D textureCaption)
+        {
+            for(int i = 0; i < 5; ++i)
```

Comment:
I've considered having colored panels drawn on top of `panelCaption` rather than overwrite the texture, but they ended up being rendered off by one pixel _after_ Unity adjusts for resolution. Seems like it was the only way to get it to work correctly in all cases

Reply:
Something to consider is that modders could replace the UI visuals here and new textures might not be read-write in import settings. And the image resolution would be different so injecting colours straight into texture maybe not the best method in the end.

I'd stick with the overlay method, even if it's off by a pixel. I find this is more of an issue in the editor than in builds. You could probably slightly enlarge the overlay in this case (even by 0.5 pixels in all directions) and it should improve things.

Reply:
How about I check for `!textureCaption.isReadable` instead? From looking at `TextureReplacement.TryImportTexture`, this seems to be what's used to tell if the mod asset is not read-write.

Also, I made `SetupCaptionColors` virtual, in case a mod wants to change how the overlay is made, in case of different resolution or other issues.

Reply:
I honestly prefer the overlay method to asking modders to add a new script for this. I'm happy to take a look as well to assist with overlay alignment if slightly enlarging doesn't help or still looks bad. There's a couple other tricks we can use here to make the overlay work.